modules/apps/frontend-data-set/frontend-data-set-admin-api/src/main/java/com/liferay/frontend/data/set/admin/model/FDSEntryCreator.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.frontend.data.set.admin.model;
+
+/**
+ * @author Daniel Sanz
+ */
+public interface FDSEntryCreator {
+
+	public void create(long companyId, long userId);
```

Comment:
Hey @dsanz @markocikos! Not sure if this is too soon to ask/out of scope: from this interface and how it's used I understand you're allowing to create `FDSEntry` based on a company-scoped `FDSView` object definition. But since I see no other arguments in the signature, are there going to be other arguments/method signatures send in also views/actions/filters? Thanks!

Reply:
You'll be able to create any relevant objects in the implementation. For example, see `FDSSimpleEntryCreator` in PR. It creates `FDSView` object entry, but it can create anything else, like actions/filters.

Oh and I'm still not sure on naming, but I doubt it'll stay `FDSEntryCreator / create`. But how it's implemented does look good to me.